### ISSUE TITLE
Strange behavior for get_noise_indices

### ISSUE THREAD

**Issue by Phildumoux** [cleanlab/cleanlab#60]
Hi,

I m trying to use cleanlab but I m not sure I m doing it right.

I have a keras model of InceptionResNetV2 on which I performed transfer learning to learn my image classification task (36 classes). The data are hand labelled and have a lot of obvious errors.

I splitted my dataset in train/valid/test. Trained the model with train, got the accuracy with valid and applied the keras model to get the probabilities on test.
I have a 65% of accuracy on train and valid/test and I perform early stopping before overfitting. I can't achieve a better accuray due to the huge noise I have in the data (tried hundreds of models and hyperparameters tuning). 
I hoped cleanlab could help me.
```

   #reading saved labels that were used during training

    labels = ap.read_labels_from_dl_log(model_id)

    #getting the indices

    label_ids = np.array(list(labels.keys()))

    #predictions are the result of model.predict(). so arrays of probabilities on the 36 classes

    psx = predictions

    log.info("PREDICTIONS  (PSX) SHAPE:" +str(psx.shape))

    #creating S

    s = np.argmax(predictions,axis=1)

    log.info("LABELS (S) SHAPE:" +str(s.shape))

    label_error_indices = get_noise_indices(
        s=s,
        psx=psx
    )
```

Problem is it return an Array of only 0.

Maybe I am mistaking somewhere?

Thanks in advance for your help.

**Comment by cgnorthcutt** [comment]
Please share the first two examples and the the shapes labels, label_ids, psx, and s. Also, please check the other resolved issues raised, some of which are like yours.

**Comment by anishathalye** [comment]
Closed due to inactivity. If this is still an issue, please provide the information requested in the comment above, and we'll do our best to help you out.
